sdk/openai/openai/src/common/interfaces.ts

```diff
@@ -1,17 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-
-import { RawHttpHeadersInput } from "@azure/core-rest-pipeline";
-
-export interface RequestOptions {
```

Comment:
This is removed because we can just import OperationOptions from core-client now.